### ISSUE TITLE
Windows terminal Access denied even with wt

### ISSUE THREAD

**Issue by dinhanhx** [microsoft/terminal#7171]
# Environment

```none
Microsoft Windows [Version 10.0.19041.421]
Windows terminal version: 1.1.2021.
```

# Steps to reproduce
Download from Microsoft Store or from Microsoft.WindowsTerminal_1.1.2021.0_8wekyb3d8bbwe.msixbundle from Github

Click launch after installation

# Expected behavior
The terminal shows up

# Actual behavior
![image](https://user-images.githubusercontent.com/38489776/89271157-84213b00-d666-11ea-82a6-ffcf9baff9fb.png)

![image](https://user-images.githubusercontent.com/38489776/89271220-969b7480-d666-11ea-85be-48cfe0c6c62e.png)

![image](https://user-images.githubusercontent.com/38489776/89271265-a5822700-d666-11ea-901e-7bbaf7b0fe67.png)

**Comment by dinhanhx** [comment]
So I have gone to this folder `C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.1.2021.0_x64__8wekyb3d8bbwe` then try double click on WindowsTerminal.exe and it works. But from other location, I couldn't

I have gone to this folder `C:\Users\DinhAnh\AppData\Local\Microsoft\WindowsApps` , and found out that wt.exe's size is 0 bytes. Double click and get the same error

**Comment by dinhanhx** [comment]
What I mean other locations such as Start Menu or Search

**Comment by zadjii-msft** [comment]
So the `wt.exe` in WindowsApps is an "app execution alias", which I believe is implemented as a "reparse point". It's basically just a symlink that points to the actual executable for the Terminal. 

It's possible that there's some hang in the Store app somewhere that's causing this - as silly as it sounds, does this go away if you reboot?

**Comment by dinhanhx** [comment]
I rebooted my computer several time, `wt.exe` still doesn't work as expected. For `WindowsTerminal.exe`, I created a shortcut of it then I move that shortcut to start menu folder. Let call that `WinTer`. `WinTer` works fine from both Search and Start menu (obviously). But the shortcut of Microsoft Store for WindowsTerminal.exe (`Windows Terminal`) can not launch.

**Comment by zadjii-msft** [comment]
That's really bizarre. 

Does [this link](answers.microsoft.com/en-us/windows/forum/all/solved-some-apps-from-microsoft-store-has/887aef4c-e912-474f-8ba2-bdd2a29d0631) from #6824 help by any chance?

**Comment by dinhanhx** [comment]
It seems you give wrong link. [The correct link](https://answers.microsoft.com/en-us/windows/forum/all/solved-some-apps-from-microsoft-store-has/887aef4c-e912-474f-8ba2-bdd2a29d0631). 

So I have tried to take ownership of these files and folders. They doesn't work at all. So I am trying to do the following:
> ===Update 3===
Updated "AppData" folder permission with "Replace all child object permission entries with inheritable permission entries from this object" and the problem is gone!!!!!

From the person who posted in the link

**Comment by dinhanhx** [comment]
@zadjii-msft It works. Thanks you sir.

**Comment by zadjii-msft** [comment]
Awesome, thanks for following up!

/dup #6824

**Comment by hrgdavor** [comment]
For me all the above did not work, but I could run wt.exe after going into "execution aliases" and the uncheck and check again alias for wt.exe. clicking that repaired my wt.exe

**Comment by mu88** [comment]
@zadjii-msft I'm encountering the same problem after installing Windows Terminal `v1.18.3181` via Chocolatey on my fresh Win10 laptop (Version 22H2, OS build `19045.3803`). I already tried fixing the permissions as described [here](https://answers.microsoft.com/en-us/windows/forum/all/solved-some-apps-from-microsoft-store-has/887aef4c-e912-474f-8ba2-bdd2a29d0631), but unfortunately, that didn't solve the issue. As a temporary workaround, I'm using the portable version, but I'd really appreciate further help to solve this issue properly 🙏🏻
